.github/workflows/node.js.yml

```diff
@@ -0,0 +1,50 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+    env:
+      TELNYX_MOCK_VERSION: 0.8.10
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
```

Comment:
we still support node v8 on this. Can we add node 8 in here?

Reply:
Done, however I'd like to move to dropping support soon and **only** supporting the actively maintained versions (https://nodejs.org/en/about/releases/) with v10 end-of-life coming in spring 2021

Reply:
agreed.